crontab

```diff
@@ -11,3 +11,5 @@
 0 19 23 4 * bundle exec rake state_file:post_deadline_reminder
 0 21 23 4 * bundle exec rake send_reject_resolution_reminder_notifications:send
 0 21 27 10 * bundle exec rake users:suspend_non_admins
+# TODO: Delete this once FYST-324 is complete.
```

Comment:
is there a story/task for this?

Reply:
It will be covered by https://codeforamerica.atlassian.net/browse/FYST-326

Reply:
Can you make sure the step is explicitly mentioned?